gridstatus/public_certificates/miso_api/intermediate_and_root.pem

```diff
@@ -1,60 +1,52 @@
-https://www.entrust.com/resources/tools/root-certificate-downloads
-# (Non‐EV SSL) CA - L1K
+# https://secure.globalsign.com/cacert/gsrsaovsslca2018.crt
```

Comment:
MISO must have changed their certificate provider from Entrust to GlobalSign.
If you run

```
openssl s_client -showcerts -connect apim.misoenergy.org:443 -servername apim.misoenergy.org </dev/null 2>/dev/null | openssl x509 -text > cert.txt
```

you can find the url for the intermediate certificate. I downloaded the intermediate cert and put it in this file. Then I googled for Global Sign's root cert and also put it in this file.